tests/integration/targets/meraki_ms_ospf/tasks/main.yml

```diff
@@ -114,10 +117,12 @@
     delegate_to: localhost
     register: enable_ospf_idempotent
 
-  - debug:
+  - name: Debug enable_ospf_idempotent
+    ansible.builtin.debug:
       var: enable_ospf_idempotent
 
-  - assert:
+  - name: Assert enable_ospf_idempotent
```

Comment:
```
TASK [meraki_ms_ospf : Debug enable_ospf_idempotent] ***************************
ok: [localhost] => {
    "enable_ospf_idempotent": {
        "changed": true,
        "data": {
            "areas": [
                {
                    "area_id": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
                    "area_name": "Office",
                    "area_type": "nssa"
                },
                {
                    "area_id": 0,
                    "area_name": "Backbone",
                    "area_type": "normal"
                }
            ],
            "dead_timer_in_seconds": 60,
            "enabled": true,
            "hello_timer_in_seconds": 20,
            "md5_authentication_enabled": true,
            "md5_authentication_key": {
                "id": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
                "passphrase": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER"
            }
        },
        "failed": false,
        "response": "OK (unknown bytes)",
        "status": 200
    }
}

TASK [meraki_ms_ospf : Assert enable_ospf_idempotent] **************************
fatal: [localhost]: FAILED! => {
    "assertion": "enable_ospf_idempotent is not changed",
    "changed": false,
    "evaluated_to": false,
    "msg": "Assertion failed"
}
```

We'll need to look into this but the other problem indeed has been resolved. Making good progress!

Reply:
@y0rune Would you please re-enable the tasks at the bottom to delete the network? That solves this problem.

Reply:
Yeah it can resolve the issue but not in the devsandbox, due to I do not have permission to `delete the network` in that organization.